db_test.go

```diff
@@ -2635,3 +2635,39 @@ func TestCompactL0OnClose(t *testing.T) {
 		}
 	})
 }
+
+func TestConcurrent(t *testing.T) {
+	dir, err := os.MkdirTemp("", "badger-concurrent-test")
+	require.NoError(t, err)
+	db, err := Open(DefaultOptions(dir))
+	require.NoError(t, err)
+
+	key := []byte("key")
+	err = db.Update(func(txn *Txn) error {
+		return txn.Set(key, []byte("value"))
+	})
+	require.NoError(t, err)
+
+	var wg sync.WaitGroup
+	for i := 0; i < 10; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			for {
+				err := db.View(func(txn *Txn) error {
+					_, err := txn.Get(key)
+					return err
+				})
+				if err != nil {
+					require.Contains(t, err.Error(), "DB Closed")
+					break
+				}
+			}
+		}()
+	}
+
+	time.Sleep(time.Second)
+	err = db.Close()
```

Comment:
nit: these two lines could be just one line `require.NoError(t, db.Close())

Reply:
ok